rlp/src/stream.rs

```diff
@@ -71,11 +82,6 @@ impl RlpStream {
 		self
 	}
 
-	/// Drain the object and return the underlying ElasticArray. Panics if it is not finished.
-	pub fn drain(self) -> Vec<u8> {
```

Comment:
This is a breaking change as I suppose?

Reply:
That's not the only breaking change (but it doesn't really matter, because this function is an alias to another public function). Every funtion that returned `Vec` now returns `BytesMut`.